### ISSUE TITLE
Tableview fields empty

### ISSUE THREAD

**Issue by Fusseldieb** [directus/app#3009]
Hi,
I've enjoying Directus so far, but one thing which I couldn't find any solution for is that when I'm "playing" around with fields and stuff, suddenly the list view doesn't show anything anymore, like below:
![image](https://user-images.githubusercontent.com/4715129/97712229-eee2e680-1a9c-11eb-9f9b-edc9d79e6ce4.png)
But if I click on it, I can clearly see the text:
![image](https://user-images.githubusercontent.com/4715129/97712324-0cb04b80-1a9d-11eb-8d3a-18d69ed5cf6b.png)

What is happening? Is this a bug or is this only some sort of misconfiguration?

Thanks in advance!

**Comment by rijkvanzanten** [comment]
I've seen this happen sometimes where a relational field was setup incorrectly which in turn makes the API try to return it as a relational field instead of raw data. Could you try checking directus_relations in the DB for anything funny?

**Comment by Fusseldieb** [comment]
> try checking directus_relations in the DB for anything funny

Yep, that solved it. It had a "loose" entry from a earlier relational "experiment" there. I deleted the suspicious entry (the entry with the field name) and it came right back :)

To future readers: If your field is NOT relational (if it's a normal text or dropdown field without any link to another table) and the field name appears inside the `directus_relations` table, it's a loose entry and you can delete it (You can access this table by enabling hidden tables inside Directus, or with your favourite SQL manager)

**Comment by rijkvanzanten** [comment]
We can leave it open as it's technically a problem in v8. I've already tackled this in the upcoming v9 by not even allowing to setup relations to non-relational fields and making sure old relations get deleted when you delete a field 🙂

**Comment by Fusseldieb** [comment]
> We can leave it open as it's technically a problem in v8. I've already tackled this in the upcoming v9 by not even allowing to setup relations to non-relational fields and making sure old relations get deleted when you delete a field 🙂

That's great news! Thank you so much for the incredibly fast response and for your help.
Directus, and everyone working on developing it (including you) are a godsend 💯
